### DISCUSSION TITLE
Images loading slowly on ipad

### DISCUSSION THREAD

**Discussion by aikoooo** [jspsych/jsPsych#2095] [Q&A]
Hello there,

I created an experiment (mnemonic similarity task) that uses a huge amount of images (about 300 images for one experiment). In the coding I made sure that all images were preloaded. When I ran the experiment on the desktop, laptop, or iPhone, there was no single problem completing the experiment. However, when running on an iPad - both safari and google chrome (which I will use for the experiment), it seems like it takes longer to load each image and sometimes image does not show up during the time (during the stimuli duration). Does anyone know how to fix this or know something that I can do for this issue?

Thank you in advance,

**Comment by jodeleeuw** [comment]
This may be the same issue as #1548. Are you using cognition.run or some other host?

**Comment by aikoooo** [comment]
Thank you for your reply! It seems like working fine now. I have another question. I am planning to do the experiment for 20 people on each ipad at the same time. I am wondering if that causes any problem. When 5ish people tried to start the experiment at the same time, it failed to preload the images. Does this happen commonly?

**Comment by jodeleeuw** [comment]
How are you hosting your experiment? If you have a web server that can handle the concurrent traffic it should be fine.
